.pre-commit-config.yaml

```diff
@@ -42,14 +42,15 @@ repos:
           - --rcfile=.pylintrc
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.0.0
+    rev: v1.8.0
     hooks:
       - id: mypy
         exclude: ^setup.py|^vendor/|^astacus/proto/
         additional_dependencies:
```

Comment:
Just in general as well, why are these not just part of requirements.texting.txt?  Some seem to be defined in multiple places.

Reply:
Because pre-commit runs in its own venv managed by its yaml file.

It's a wontfix: https://github.com/pre-commit/pre-commit/issues/730